File: coforge 2024_may/corejava/src/Calculator.java
public class Calculator {

	static int addition(int a, int b) {
		return a + b;
	}

	static int subtraction(int a, int b) {
		return a - b;
	}

	static int multiplication(int a, int b) {
		return a * b;
	}

	static int division(int a, int b) {
		return a / b;
	}

	public static void main(String[] args) {
		int a = 10, b = 5;
		System.out.println("Addition of " + a + " and " + b + " is " + addition(a, b));
		System.out.println("Subtraction of " + a + " and " + b + " is " + subtraction(a, b));
		System.out.println("Multiplication of " + a + " and " + b + " is " + multiplication(a, b));
		System.out.println("Division of " + a + " and " + b + " is " + division(a, b));
	}

}

File: web/angular gl/app1/src/app/calculator/calculator.ts
import { CommonModule } from '@angular/common';
import { Component, Input } from '@angular/core';
import { FormsModule } from '@angular/forms';

@Component({
  selector: 'app-calculator',
  imports: [CommonModule, FormsModule],
  templateUrl: './calculator.html',
  styleUrl: './calculator.css',
})
export class Calculator {
  @Input() n1: number = 0;
  @Input() n2: number = 0;
  @Input() n3: number = 0;

  addition() {
    this.n3 = this.n1 + this.n2;
  }
  subtraction() {
    this.n3 = this.n1 - this.n2;
  }
  mutiplication() {
    this.n3 = this.n1 * this.n2;
  }
  division() {
    this.n3 = this.n1 / this.n2;
  }


  hello() {
    alert("hello called ");
  }


}
